src/mrpro/data/_CsmData.py

```diff
@@ -27,7 +27,9 @@ class CsmData(QData):
 
     @staticmethod
     def _iterative_walsh_csm(
-        coil_images: torch.Tensor, smoothing_width: SpatialDimension[int], niter: int
+        coil_images: torch.Tensor,
+        smoothing_width: SpatialDimension[int],
```

Comment:
maybe mention in docstring that int means same smoothing along all non-singleton k dimensions?

Reply:
there should be a test calling it with int and one calling it with spatialdimension

Reply:
added `test_CsmData_interactive_Walsh_smoothing_width` testing equality of csm data using calling iterative walsh with SpatialDimension and int